cmd/fluent-watcher/fluentd/main.go

```diff
@@ -177,7 +177,7 @@ func newWatcher(poll bool, interval time.Duration) (filenotify.FileWatcher, erro
 
 // Inspired by https://github.com/jimmidyson/configmap-reload
 func isValidEvent(event fsnotify.Event) bool {
-	return event.Op == fsnotify.Create || event.Op == fsnotify.Write
+	return event.Op == fsnotify.Rename
```

Comment:
Does rename contain the op of Create or Write? Why fluent bit no need to modify?

Reply:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/18525465/167249505-047fb8e1-e380-4a07-a0db-7383296d6c93.png">

Reply:
does this means fluentbit agent also need to update the event here?

Reply:
It seems to be the mechanism of statefulset that will create a soft link, check this out.
https://github.com/kubernetes/kubernetes/pull/86869